packages/js/src/Modules/Verto/util/tests/helpers.test.ts

```diff
@@ -0,0 +1,63 @@
+import { getGatewayState } from '../helpers';
+import { IRequestRPC, IResponseRPC } from '../interfaces';
+
+describe('getGatewayState', () => {
+  it('it should return empty string if server did not send the gateway state in the request', () => {
+    const msg: IRequestRPC = {
+      id: '',
+      method: '',
+      params: '',
+    };
+
+    const state = getGatewayState(msg);
+    expect(state).toBeFalsy();
+  });
+  it('it should return empty string if server did not send the gateway state in the response', () => {
```

Comment:
```suggestion
  it('response: it should return empty string if server did not send the gateway state', () => {
```

Reply:
prefixing `request` or `response` makes it a lot more readable since they are all inside the same block. You can also move the tests inside a `describe('request')` and `describe('response')` block.